db/migrations/Reference.php

```diff
@@ -17,13 +18,14 @@ protected function referencesUser(Blueprint $table, $setPrimary = true)
 
     /**
      * @param Blueprint $table
-     * @param string    $targetTable
-     * @param string    $fromColumn
-     * @param bool      $setPrimary
+     * @param string $targetTable
+     * @param string $fromColumn
+     * @param bool $setPrimary
+     * @return ColumnDefinition
```

Comment:
Hmm, i think that using spaces is better readable but thats another point for a code style discussion ;)

Reply:
I think that as well ✓